viewer/packages/cad-parsers/src/cad/primitiveGeometries.ts

```diff
@@ -121,6 +121,7 @@ export const { torusLodGeometries, torusGeometryBoundingBox } = (() => {
 
   return {
     torusLodGeometries,
+    // TODO, hflatval: I believe this is wrong, torusLodGeometries are just planes near the origin
```

Comment:
```suggestion
    // TODO 2021-12-10 hflatval: I believe this is wrong, torusLodGeometries are just planes near the origin
```
I love TODOs with dates (if they get one year old they will never be fixed so then we can consider to remove them)

Reply:
This is a plain bug - so more than a TODO. We should either fix or remove support for filtering of torus for now.